evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -823,3 +823,14 @@ def test_explain_predictions_stacked_ensemble(problem_type, dummy_stacked_ensemb
 
     with pytest.raises(ValueError, match="Cannot explain predictions for a stacked ensemble pipeline"):
         explain_predictions_best_worst(pipeline, X, y)
+
+
+@pytest.mark.parametrize("estimator", ["Logistic Regression Classifier", "Decision Tree Classifier", "Extra Trees Classifier"])
```

Comment:
Rather than specify the estimators to test, it could be a good idea to pass in all of the estimators using all_estimators and then skipping the estimators that don't work :)